metadata-service/war/src/main/resources/boot/groups.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "urn": "urn:li:corpgroup:organization-admins",
+    "info": {
+      "displayName": "Organization Admins",
+      "description": "Top level DataHub administrators. They have ALL privileges.",
+      "admins": [],
+      "members": [],
+      "groups": []
+    }
+  },
+  {
+    "urn": "urn:li:corpgroup:metadata-stewards",
+    "info": {
+      "displayName": "Metadata Stewards",
+      "description": "Overseers of all metadata assets across DataHub.",
+      "admins": [],
+      "members": [],
+      "groups": []
+    }
+  },
+  {
+    "urn": "urn:li:corpgroup:metadata-readers",
+    "info": {
+      "displayName": "Metadata Readers",
+      "description": "Users in this group have the ability to read metadata, but cannot change it by default.",
+      "admins": [],
+      "members": [],
+      "groups": []
+    }
+  },
+  {
+    "urn": "urn:li:corpgroup:access-admins",
+    "info": {
+      "displayName": "Access Admins",
+      "description": "Administrators for managing users, groups, and security-related features.",
```

Comment:
Let's make sure the content of the description is extremely consistent. Ideally for each you just say what they have access to do!

Reply:
Made them more consistent.